docs/advanced-topics/checksum-support.rst

```diff
@@ -37,6 +50,22 @@ Descriptions for the two endpoints of note are as follows:
 
 Just like the file endpoints, the id parameter used in the endpoints above can be found on an entry's public page; underneath the Info tab, look for the bolded words **TRS**.
 
+To verify a checksum as reported by the Dockstore API matches what you download from the Docker registry first find the checksum
+and image path using one of the above methods for the image you would like to verify. Then download the image using the
+Docker CLI client.
+
+::
+
+    docker pull quay.io/briandoconnor/dockstore-tool-md5sum:1.0.4
+
+When the download has completed a Digest is provided in the terminal output. This should match the checksum provided
+by the Dockstore API.
+
+Verifying the image checksum can give you better guarantees the image has not changed since it was published to Dockstore.
+However, in some cases the image checksum may diverge, for example, if the image was defined in a git branch that has since
```

Comment:
Can also mention/link that if an image is defined off of a branch, Docker Hub is more likely to delete it when they do cleanup. A digest corresponding to a tagged version is probably better for the longevity